goinstant/goinstant.go

```diff
@@ -30,7 +48,10 @@ var customOptions = customOption{
 func stopContainer() {
 	commandSlice := []string{"stop", "instant-openhie"}
 	suppressErrors := []string{"Error response from daemon: No such container: instant-openhie"}
-	runCommand("docker", suppressErrors, commandSlice...)
+	_, err := runCommand("docker", suppressErrors, commandSlice...)
+	if err != nil {
+		gracefulPanic(err, "")
```

Comment:
This call to gracefulPanic results in an infinite loop on error. I fixed this in a separate PR.